awx/main/management/commands/host_metric.py

```diff
@@ -0,0 +1,66 @@
+from django.core.management.base import BaseCommand
+from dateutil import parser
+from datetime import timezone
+from awx.main.models.inventory import HostMetric
+import json
+
+
+class Command(BaseCommand):
+
+    help = 'This is for offline licensing usage'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--since', dest='since', action='store', help='Start date for data')
+        parser.add_argument('--until', dest='until', action='store', help='End date for data')
+        parser.add_argument('--json', action='store_true', help='Select output as JSON')
+
+    def handle(self, *args, **options):
+        opt_since = options.get('since')
+        opt_until = options.get('until')
+
+        # user gives no arguments
+        if opt_since is None and opt_until is None:
+            print("No Arguments received")
+            return None
+
+        since = None
+        if opt_since:
+            since = parser.parse(opt_since)
+            if since.tzinfo is None:
+                since = since.replace(tzinfo=timezone.utc)
+
+        until = None
+        if opt_until:
+            until = parser.parse(opt_until)
+            if until.tzinfo is None:
+                until = until.replace(tzinfo=timezone.utc)
+
+        filter_kwargs = {}
+        if since is not None:
+            filter_kwargs['last_automation__gte'] = since
+        if until is not None:
+            filter_kwargs['last_automation__lte'] = until
+
+        result = HostMetric.objects.filter(**filter_kwargs)
+
+        # if --json flag is set, output the result in json format
+        if options['json']:
+            list_of_queryset = list(result.values('hostname', 'first_automation', 'last_automation'))
+            for item in list_of_queryset:
+                item['first_automation'] = item['first_automation'].strftime('%Y-%m-%d')
```

Comment:
If you're doing this because the date objects aren't JSON-serializable, there's another way to accomplish this by providing `json.dumps()` a hint about how to serialize dates (that way you don't have to loop over this data structure and replace the values in-place):

```python
from django.core.serializers.json import DjangoJSONEncoder
if options['json']:
    print(json.dumps(
        result.values('hostname', 'first_automation', 'last_automation')
    ), cls=DjangoJSONEncoder)
```

Reply:
I am doing this because I was wondering Customer would want that second level granularity. :sweat_smile:.
I think he would look at dates ```"last_automation": "2021-04-13"```  rather than ```"last_automation": "2021-04-13T16:08:56.841Z"``` in the JSON

Reply:
I have made the changes but let me know if i need to revert back.